__docs__/wonder-blocks-accordion/accordion.stories.tsx

```diff
@@ -517,6 +522,57 @@ export const LongSections: StoryComponentType = {
     },
 };
 
+/**
+ * This is an example of an Accordion with a dropdown within each section.
+ * This demonstrates how the accordion keyboard interactions do not interfere
+ * with the dropdown's keyboard interactions.
+ */
```

Comment:
**suggestion:** Could you please move this comment block before the `export const WithDropdown` line? That way SB will be able to include that in the docs page.